new/components/nginx/config/rest-location.cloudify

```diff
@@ -1,4 +1,4 @@
-location ~ ^/api/v2/(blueprints|executions|deployments|nodes|events|search|status|provider|api|node-instances|version|evaluate|deployment-modifications|tokens) {
+location ~ ^/api/v2/(blueprints|snapshots|executions|deployments|nodes|events|search|status|provider|api|node-instances|version|evaluate|deployment-modifications|tokens) {
```

Comment:
@m1keil, @idanmo, as a general comment, I think we should discuss this, but snapshot is a misnomer. We're not taking snapshots, but rather a full backup. A snapshot is a relative point in time. When we do provide incremental backups (and then, we'll probably call them snapshots), this will bite us in the ass API-wise.